config/env/development.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
     }
   },
   app: {
-    title: defaultEnvConfig.app.title + ' - Development Environment'
+    title: defaultEnvConfig.app.title + ' - Development Environment',
+    domain: 'localhost:3000'
```

Comment:
Why is the port required here? Is it because you're using `req.headers.hostname`?